llm/README.md

```diff
@@ -0,0 +1,18 @@
+# Data LLM
+
+This directory contains code to train, evaluate and run inference on LLMs
+integrated with Data Commons.
```

Comment:
I guess for now there is only inference but not "train and evaluate"?

Reply:
Updated.